rfcs/text/0022-remove-process-ppid.md

```diff
@@ -15,50 +15,82 @@ Stage 0: Provide a high level summary of the premise of these changes. Briefly d
 
 The `process.ppid` field has been part of ECS since 1.0.0. The `process.parent.*` fields were later introduced in ECS 1.3.0, including `process.parent.pid`. Both fields serve the same purpose: capture the process identifier (PID) of a process' parent.
 
-There's no need to have two fields capturing the same value, and to avoid duplication or user confusion, one of the fields should be removed. With all of the parent process fields are now included in ECS with the `process.parent.*` nesting, `process.ppid` could be deprecated and then later removed.
+There's no need to have two fields to capture the same value and to avoid unneeded duplication and confusion, one of the fields should be removed. All of the parent process fields are now included in ECS with the `process.parent.*` nesting, and `process.ppid` should be deprecated and later removed.
 
 ## Fields
 
-<!--
-Stage 1: Describe at a high level how this change affects fields. Include new or updated yml field definitions for all of the essential fields in this draft. While not exhaustive, the fields documented here should be comprehensive enough to deeply evaluate the technical considerations of this change. The goal here is to validate the technical details for all essential fields and to provide a basis for adding experimental field definitions to the schema. Use GitHub code blocks with yml syntax formatting.
--->
+Removing `process.ppid` will take place in two steps:
 
-### Timeline
+1. ECS `1.x`: Indicate that `process.ppid` is deprecated in the fields description in an upcoming ECS minor release. Producers and consumers of `process.ppid` should transition to using `process.parent.id` instead.
```

Comment:
@ebeahan just a heads up there's a couple places in the RFC that we'll want to switch to `process.parent.pid` instead of `process.parent.id` 👍

Reply:
> > * Security App - We have process lineage for Analyzer.
> 
> Within the process analyzer we don't use it to layout the graph (we use `entity_id` for that). I don't _think_ we use it anywhere else except maybe displaying the value in the side panel. @michaelolo24 or @kqualters-elastic is that accurate?

Yea, I don't believe it's used anywhere, just displayed.